adding kibana_server role specification and explanation #4094 (#7066)
opensearch-project/documentation-website: _security/multi-tenancy/multi-tenancy-config.md

| `multitenancy_enabled` | Enable or disable multi-tenancy. Default is `true`. |
| `private_tenant_enabled` | Enable or disable the private tenant. Default is `true`. |
| `default_tenant` | Use to set the tenant that is available when users log in. |

> | `server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`. If a different user is configured, make sure this user is mapped to `kibana_server` role via `role_mappings.yml` file, to give the appropriate permissions listed in [kibana_server role details]({{site.url}}{{site.baseurl}}/security/multi-tenancy/multi-tenancy-config/#kibana_server-role-details) |

Reviewer: ```suggestion
| `server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`. If a different user is configured, then make sure such user is mapped to the `kibana_server` role via the `role_mappings.yml` file to give that user the appropriate permissions listed in [kibana_server role details]({{site.url}}{{site.baseurl}}/security/multi-tenancy/multi-tenancy-config/#kibana_server-role-details). |
```

Reviewer 2: Edited suggestion

{: .tip }

To back up your OpenSearch Dashboards data, [take a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/) of all tenant indexes using an index pattern such as `.kibana*`.

> ## `kibana_server` role details

Reviewer 2: ```suggestion
## `kibana_server` role
```

> `kibana_server` role is used by OpenSearch Dashboards to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role via `role_mappings.yml`. You can view the full list of permissions assigned to this role by running `GET` request on `_plugins/_security/api/roles/kibana_server` API using admin certificate, key and certificate authority file.

Reviewer: ```suggestion
OpenSearch Dashboards uses the`kibana_server` role to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role through the `role_mappings.yml` file. You can view the full list of permissions assigned to this role by running the GET request on `_plugins/_security/api/roles/kibana_server` API using the admin certificate, key, and certificate authority file.
```

Reviewer 2: Edited suggestion. Is "on the...API" the correct wording? The last sentence is a bit of a run-on.

> Following is the full list of permissions assigned to this role:

Reviewer: ```suggestion
The following list includes the permissions assigned to this role:
```
